src/Outsystems.SetupTools/Lib/PlatformSetup.ps1

```diff
@@ -166,12 +239,12 @@ function InstallDotNetCore([string]$Sources)
 {
     if($Sources)
     {
-        $installer = "$Sources\DotNetCore_2_WindowsHosting.exe"
+        $installer = "$Sources\DotNetCore_WindowsHosting.exe"
         LogMessage -Function $($MyInvocation.Mycommand) -Phase 1 -Stream 2 -Message "Using local file: $installer"
     }
     else
     {
-        $installer = "$ENV:TEMP\DotNetCore_2_WindowsHosting.exe"
+        $installer = "$ENV:TEMP\DotNetCore_WindowsHosting.exe"
```

Comment:
(same as above) We should move this to a constant in src/Outsystems.SetupTools/Lib/Constants.ps1